common/src/test/java/org/apache/uniffle/common/RemoteStorageInfoTest.java

```diff
@@ -119,4 +119,14 @@ public void testHashCode() {
     RemoteStorageInfo info1 = new RemoteStorageInfo(TEST_PATH, CONF_STRING);
     assertEquals(info.hashCode(), info1.hashCode());
   }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"k1=v1\\,v11,k2=v2\\,v22"})
```

Comment:
Em.. You should answer the question
`Is it a user-face change`?
I think it's yes. If it's a user-face change, we should add document and see whether it is a compatible change.

Reply:
Is it more reasonable?
```
xxx,xxx;
xxx,xxx;
```

Reply:
Yes, we should add document. I will do it later...
Do you means that?
```
key1=value1,value11;key2=value2 
```
or 
```
cluster;key1=value1,value11;key2=value2
```
I am not sure about it. Because I think ';' may include in config value; For example:

```
    <property>
        <name>hadoop.user.group.static.mapping.overrides</name>
        <value>dr.who=;hdfs=supergroup,hdfs,hadoop</value>
    </property>
```

Reply:
> Yes, we should add document. I will do it later... Do you means that?
> 
> ```
> key1=value1,value11;key2=value2 
> ```
> 
> or
> 
> ```
> cluster;key1=value1,value11;key2=value2
> ```
> 
> I am not sure about it. Because I think ';' may include in config value; For example:
> 
> ```
>     <property>
>         <name>hadoop.user.group.static.mapping.overrides</name>
>         <value>dr.who=;hdfs=supergroup,hdfs,hadoop</value>
>     </property>
> ```

Option 2 is better. Yes, `;` is also a reasonable splitter in the configuration, so we choose it.  It's ok only if `dfs.ha.namenodes.ns-fed` won't use it.

Reply:
@jerqi 
';' have been used.
It is for 'rss.coordinator.remote.storage.path', the config value looks like:

```
rss.coordinator.remote.storage.path clustera,k1=v1,k2=v2;clusterb,k1=v1,k2=v2;clusterc,k1=v1,k2=v2;
```

Reply:
@zuston @advancedxy Do you have better solution? If we don't have better solution, it's also acceptable. But we should add doc to explain how to use it.

Reply:
OK, wait for @zuston @advancedxy 's suggestion.
If all feel unacceptable, I think a better way is that extract a new config file for remote storage, but it is more complex for user.

Reply:
@LuciferYang Do you have another suggestion?

Reply:
> Is this an Improvement or a bug fix? In #1110, I saw that its title is an `[Improvement]`, but in the description, `Describe the bug` is filled in, which seems a bit ambiguous.

I think this is improvement.

Reply:
OK, suggest modifying #1110, to make it more clearer.”

Reply:
@zhengchenyu Could we split the
```
rss.coordinator.remote.storage.path clustera,k1=v1,k2=v2;clusterb,k1=v1,k2=v2;clusterc,k1=v1,k2=v2;
```
into
```
rss.coordinator.remote.storage.path.clustera k1=v1,v2;k2=v3
rss.coordinator.remote.storage.path.clusterb k1=v1;k2=v2
rss.coordinator.remote.storage.path.clusterc k1=v1;k2=v2
```
It seems more reasonable.

Reply:
+1, We can add a new configuration and parsing way to meet new requirements, while keeping the original way for compatibility.

Reply:
> @zhengchenyu Could we split the
> 
> ```
> rss.coordinator.remote.storage.path clustera,k1=v1,k2=v2;clusterb,k1=v1,k2=v2;clusterc,k1=v1,k2=v2;
> ```
> 
> into
> 
> ```
> rss.coordinator.remote.storage.path.clustera k1=v1,v2;k2=v3
> rss.coordinator.remote.storage.path.clusterb k1=v1;k2=v2
> rss.coordinator.remote.storage.path.clusterc k1=v1;k2=v2
> ```
> 
> It seems more reasonable.

@jerqi That's a great idea!  +1, thanks for your suggestion.

Reply:
> +1, We can add a new configuration and parsing way to meet new requirements, while keeping the original way for compatibility.

@LuciferYang I can not catch you mean. I think we can not keep compatible.

Reply:
> > +1, We can add a new configuration and parsing way to meet new requirements, while keeping the original way for compatibility.
> 
> @LuciferYang I can not catch you mean. I think we can not keep compatible.

We add a new configuration option like 
```
rss.coordinator.remote.storage.conf
```
We don't change the old config option to keep compatible.

Reply:
> > > +1, We can add a new configuration and parsing way to meet new requirements, while keeping the original way for compatibility.
> > 
> > 
> > @LuciferYang I can not catch you mean. I think we can not keep compatible.
> 
> We add a new configuration option like
> 
> ```
> rss.coordinator.remote.storage.conf
> ```
> 
> We don't change the old config option to keep compatible.

I got it! Thanks very much!